ros2/test.bzl

```diff
@@ -57,3 +62,21 @@ def ros2_test(name, nodes, launch_file, deps = None, data = None, **kwargs):
         ] + deps,
         **kwargs
     )
+
+def _ros2_launch_pytest_test(name, nodes, launch_file, deps, data, **kwargs):
+    py_pytest_test(
+        name = name,
+        srcs = [launch_file],
+        data = nodes + data,
+        deps = [
+            "@ros2_launch//:launch_pytest",
+        ] + deps,
+        args = [
+            "-p",
+            "launch_pytest.plugin",
+        ],
```

Comment:
You can add this directly in the runner template file, no need to expose it here.